### ISSUE TITLE
Meeting Sept. 2014

### ISSUE THREAD

**Issue by themasch** [loldevs/leaguespec#25]
After our meeting last sunday i thought it would be a great idea to do these meetings at least once a month to bring everyone whos interested in sync with the current status of this project. 
I'd like to use this issue to find a new date for the next meeting and collect topics to discuss besides general status updates. 

The main problem on finding a date might be the different timezones ppl are living in. Because of that I'd suggest to do these meetings on a weekend so we can schedule them more freely. 

So I'd suggest the 5., 6. or 7. Sept. for the next meeting. Any other suggestions?
Do you guys (and girls?) even want regular meetings

**Comment by jaagupkymmel** [comment]
We shouldn't have a strict "once per month" schedule, rather a more flexible one. Other than that, I'm in.

**Comment by themasch** [comment]
so just a "meeting when needed"?

**Comment by vafrederico** [comment]
I would say at least a meeting per month, doesn't need to be strict to a
date, only to catch things up.

Att,
Vinícius Andrade Frederico
E-mail: vafrederico@gmail.com

2014-08-05 6:29 GMT-03:00 Mark Schmale notifications@github.com:

**Comment by Divi** [comment]
Ok, but create an issue (and an email will be sent to the team) or a wiki page before each meeting, because I didn't know for the last Sunday.

**Comment by jaagupkymmel** [comment]
Certainly. This meeting was actually supposed to take place this Saturday, but I misclicked and selected the wrong date in Doodle. There was already a fair bit of people online at the time it was scheduled, so I decided we will do it anyway.

**Comment by themasch** [comment]
@Divi thats why i created that issue ;)

**Comment by jaagupkymmel** [comment]
So @themasch, can you organize a meeting?

**Comment by themasch** [comment]
I can do. Theres a lot of progress in the libOL thing and we got huge steps in the leaguespec wiki, both thanks to @satan6. I'll start a doodle soon.

**Comment by themasch** [comment]
Okay, does anyone got suggestions for dates? 
I think a saturday might be best so the 6. or the 13. might be possible. 
What time would you suggest?

**Comment by jaagupkymmel** [comment]
You can just add multiple times in doodle.

**Comment by themasch** [comment]
Yeah, i know that. I just wanted to get rough range so i dont need to add all possible times :D 

Okay: Heres the Doodle: http://doodle.com/bbht474rvwgsucyz

**Comment by themasch** [comment]
Okay, I think we should do the meeting on Sept. 13. 19.00 CEST (UTC+2)

**Comment by Divi** [comment]
:+1:

**Comment by themasch** [comment]
I added a list of topics to the wiki. if you find something missing, please add it: 
https://github.com/loldevs/leaguespec/wiki/Meeting-Summaries#saturday-september-13-2014

We also need some ppl for some topics, if you feel responsible, add your name ;)

**Comment by Divi** [comment]
I cannot be there finally, could you write down a summary after the meeting please ? Thanks in advance :)
